src/test/java/io/rcktapp/api/handler/sql/SqlGetHandlerTest.java

```diff
@@ -0,0 +1,125 @@
+package io.rcktapp.api.handler.sql;
+
+import io.forty11.j.J;
+import io.forty11.sql.Sql;
+import io.forty11.web.Url;
+import io.rcktapp.api.*;
+
+import io.rcktapp.api.Collection;
+import io.rcktapp.api.service.Service;
+
+import io.rcktapp.rql.Replacer;
+import io.rcktapp.rql.Rql;
+import io.rcktapp.rql.Stmt;
+import io.rcktapp.rql.sql.SqlRql;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.*;
+
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.sql.Connection;
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+public class SqlGetHandlerTest {
+
+    @Mock
+    Service service;
+
+    @Mock
+    Api api;
+
+    @Mock
+    Endpoint endpoint;
+
+    @Mock
+    Action action;
+
+    @Mock
+    Chain chain;
+
+    @Mock
+    Response response;
+
+    @Mock
+    Collection collection;
+
+    @Mock
+    Entity entity;
+
+    @Mock
+    Table table;
+
+    @Mock
+    Attribute attribute;
+
+    @Mock
+    Stmt statement;
+
+    @Mock
+    SqlDb sqlDb;
+
+    @Mock
+    SqlRql sqlRql;
+
+    @Mock
+    Connection connection;
+
+    @Mock
+    SqlGetHandler handler;
+
+    @Captor
+    ArgumentCaptor<List> queryParametersCaptor;
+
+    @Test
+    public void testSqlGetHandler_queryParametersAreProperlyPassedToQueryObjectsMethod() throws Exception {
+
+        Url url = new Url("http://localhost:8080/api/lift/us/plugins/updatexml(rand(),concat(CHAR(126),user(),CHAR(126)),null)");
```

Comment:
Should there be another test too that also includes actual parameters?

http://localhost:8080/api/lift/us/plugins/updatexml(rand(),concat(CHAR(126),user(),CHAR(126)),null)?name1=asdf&name2=qwer